### ISSUE TITLE
nodejs with version after 15.0.0 can't work with nvm

### ISSUE THREAD

**Issue by Vixb1122** [coreybutler/nvm-windows#579]
**NVM4W version:** 1.1.7

### My Environment

Windows 10 with Chinese

### Expected Behavior

use 15.0.0 and manage global package with npm

### Actual Behavior

use 15.0.0 and install global package, but got empty list after run `npm ls -g`;

### Steps to reproduce the problem:

1. run `nvm install 15.0.0`
2. run `nvm use 15.0.0`
3. run `npm ls -g` and get empty list
4. run `npm i yarn -g`
5. run `npm ls -g` and get empty list as before

### Output
```
C:\Users\Test> nvm install 15.0.1
Downloading node.js version 15.0.1 (64-bit)...
Complete
Creating C:\Users\Test\AppData\Roaming\nvm\temp

Downloading npm version 7.0.3... Complete
Installing npm v7.0.3...

Installation complete. If you want to use this version, type

nvm use 15.0.1
C:\Users\Test> nvm use 15.0.1
Now using node v15.0.1 (64-bit)
C:\Users\Test> npm ls -g
C:\Program Files\nodejs -> C:\Users\Test\AppData\Roaming\nvm\v15.0.1
`-- (empty)

C:\Users\Test> npm i -g yarn

added 1 package in 1s
C:\Users\Test> npm ls -g
C:\Program Files\nodejs -> C:\Users\Test\AppData\Roaming\nvm\v15.0.1
`-- (empty)
```

**Comment by coreybutler** [comment]
I'm able to recreate this, but I do not know why it's happening. The `node_modules` directories are in the appropriate locations, which is evident by the fact you can run yarn after the install. Additionally, if you run `npm ls -g` from `C:\Users\Test\AppData\Roaming\nvm\v15.0.1`, it will work. 

![image](https://user-images.githubusercontent.com/770982/98199076-0a604e00-1ef0-11eb-8e8b-f503a741240d.png)

The only thing I can think of is the npm `ls` command may not be respecting the symlink, i.e. `C:\Program Files\nodejs -> C:\Users\Test\AppData\Roaming\nvm\v15.0.1`. 

Bottom line, I don't think this is something NVM4W can fix, so I'm closing as `won't fix`. I find it very strange npm would respect symlinks for executing commands and not for listing them, so I suspect this is an upstream issue. If anyone finds a workaround (or can point me to the source of the problem in npm), I'd be happy to take another look at it.

**Comment by NbrTDB** [comment]
I too would be interested in any workarounds for this.

**Comment by Vixb1122** [comment]
@NbrTDB 
Subscribe https://github.com/npm/cli/issues/2229
